### ISSUE TITLE
Forknote web wallet (unofficial)

### ISSUE THREAD

**Issue by streamcoindev** [forknote/forknote#103]
https://github.com/streamcoindev/ForkNote-webwallet

Works with all forknote coins

**Comment by xlnd47** [comment]
This is lit man! GJ

**Comment by xlnd47** [comment]
![image](https://user-images.githubusercontent.com/26673889/37592280-7f05c302-2b6e-11e8-9f7a-390b5b2edfeb.png)
![image](https://user-images.githubusercontent.com/26673889/37592313-946b464a-2b6e-11e8-9f4c-1bae0647f411.png)
![image](https://user-images.githubusercontent.com/26673889/37592329-9e01fef6-2b6e-11e8-8bc7-055946a4ce1b.png)
So first one is my php on my website, second one is index page of my website, 3th one is response i get if i use the curl on onlinecurl.com. Can you maybe tell me what i am doin wrong here?

**Comment by xlnd47** [comment]
I cant connect my website to my walletd

**Comment by streamcoindev** [comment]
First of all did you set walletd to run on port 9090 ? second why do you not use 127.0.0.1 instead of your external ip or is your walletd running on another server ? so when you use curlonline.com it works but when you use php it doesnt ? i will check my code wtih yours

**Comment by streamcoindev** [comment]
also you have streamcoind running right?

**Comment by streamcoindev** [comment]
also what do you see when you echo result 8?

**Comment by streamcoindev** [comment]
just started everything up again for me it still works fine 
http://storage5.static.itmages.com/i/18/0319/h_1521470057_3280378_b5282d1918.png

**Comment by xlnd47** [comment]
Yes walletd is running on port 9090, indeed, website is on another ip then walletd. Forknoted is also running yes, but echo result8 is 0.

**Comment by streamcoindev** [comment]
can you type out your wallet address?

**Comment by streamcoindev** [comment]
cause when i try to connect it says bad address but it connects how big is you varchar in your mysql table of column address?

**Comment by xlnd47** [comment]
varchar97
2DfEPcH1QuehiuPFHwooa9Rve8t9eG7qCAmGEn1RrjeQ4DTdA7Dw3W22Sq7nZoy9nKjQ8VHp1JLAzYdDfsG1YWem7i7756e

**Comment by xlnd47** [comment]
ok found whats wrong, my webhost blocks almost every port, 9090 included.

**Comment by xlnd47** [comment]
![image](https://user-images.githubusercontent.com/26673889/37646365-c5d5918e-2c29-11e8-91aa-8b8a4828610d.png)
Now its working

**Comment by streamcoindev** [comment]
@xlnd47 thats nice !

**Comment by xlnd47** [comment]
@streamcoindev Does it work for you to withdraw coins to another wallet?

**Comment by streamcoindev** [comment]
@xlnd47 yes i tested this multiple times why does it not work for you ?

**Comment by xlnd47** [comment]
curl -X POST -i -H "Accept: application/json" -d '{"params": {"anonymity":0.1, "fee":1000000000,"transfers":[{"amount":100000000000,"address":"2DfEPcH1QuehiuPFHwooa9Rve8t9eG7qCAmGEn1RrjeQ4DTdA7Dw3W22Sq7nZoy9nKjQ8VHp1JLAzYdDfsG1YWem7i7756e"}], "changeAddress": "2JbMrwFEPdrFQ1tQne8ic6VdLBZmifFpuSnSBBKiMtXRU2ge3jfmv2XiaD9vRNd5rn9y47zyXZXXqD6icQ4DBkyd7iFHgv4"},"jsonrpc": "2.0", "id": "test","method":"sendTransaction"}' http://107.175.60.139:9090/json_rpc

I tried this curl command, but doesnt work, do you maybe know where i made a mistake?

**Comment by xlnd47** [comment]
![image](https://user-images.githubusercontent.com/26673889/37725127-849dd456-2d32-11e8-9372-8d05244b0914.png)
On http://forknote.net/documentation/rpc-wallet/rpc-wallet-curl-examples/ they say its not working

**Comment by streamcoindev** [comment]
it should work with my form ?
Doesnt it work with my form >?

**Comment by xlnd47** [comment]
Yes now its working with your form!

**Comment by vitorgamer58** [comment]
interessante
